.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Pull Request
+about: Open a Pull Request for this project
+title: ''
+labels: 'type: pull request'
+assignees: ''
+---
+
```

Comment:
```suggestion
```
I believe these aren't needed for the PR template

Reply:
Sorry, I missed your earlier comment!